apps/external-devices/device-descriptors/5DT_Data_Glove_14_Utra.json

```diff
@@ -0,0 +1,13 @@
+{
+    "deviceVendor": "Fifth Dimension Technologies",
+    "deviceName": "5DT Data Glove 14 Utra",
```

Comment:
spelling?

Also, I don't know exactly the mapping for the 14 ultras, but I think I know the joint mapping for the 5 ultras, not sure how important that is.